PerfNext/public/lib/js/util.js

```diff
@@ -246,6 +248,127 @@ function getEspressoPlatform() {
     }
     return platform;
 }
+/* Get the OS, cputype and bits to check against the master machine list */
+function getCPUInfo() {
+	console.log('Entering getCPUInfo()');
+	
+    var platform = document.getElementById('dtextlabel').innerText;
+    console.log('platform: '+platform);
+    
+    var cpuInfo = {};
+    
+    switch (platform) {
+    
+    case 'Windows Intel x86 32 bit':
+    	cpuInfo['cpuType'] = 'x86';
```

Comment:
Instead of using `cpuInfo` every time, you could do something like this: ```cpuInfo = {cpuType: 'x86', bits: '32',  OS: 'Windows'};``` in just one line.